packages/rich-text-from-markdown/src/index.ts

```diff
@@ -232,6 +232,13 @@ async function mdToRichTextNode(
   fallback: FallbackResolver,
   appliedMarksTypes: string[] = [],
 ): Promise<Node[]> {
+  // By default <br/> is parsed as html node, causing it to be stripped out.
+  // We need to convert it manually in order to support it
+  if (node.type === 'html' && node.value === '<br/>') {
```

Comment:
I would also checked for `<br>` and `<br />` (with space).

Reply:
Good point, I'll write a regex for that